docs/wall.html

```diff
@@ -0,0 +1,225 @@
+    <!-- materialize -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/js/materialize.min.js"></script>
+    <!-- page -->
+    <script src="js/libs/countup.min.js"></script>
+    <link type="text/css" rel="stylesheet" href="css/wall.css"/>
+    <script type="module" src="js/wall.js"></script>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+
+<body>
+    <nav>
+        <div class="nav-wrapper">
+            <a href="#" class="brand-logo"><img src="resources/logo240.png" class="logo-img"/><span
+                    class="hide-on-small-only">emeInvestor bot</span></a>
+            <a href="#" data-target="mobile-sidebar" class="sidenav-trigger"><i
+                    class="material-icons">menu</i></a>
+
+            <ul id="nav-mobile" class="right hide-on-med-and-down">
+                <li>
+                  <div class="input-field">
+                    <input id="investor-username" class="custom-dark-search" type="search" placeholder="search user" >
+                    <label class="label-icon" for="search-nav"><i class="material-icons">search</i></label>
+                    <i class="material-icons" id="investor-username-search">send</i>
+                  </div>
+                </li>
+                <!-- <li><a href="foo.html">overview</a></li> -->
+                <li><a href="./index.html">home</a></li>
+                <li><a class="grey-text" >leaderboard</a></li>
+                <li>
+                  <a href="https://github.com/MemeInvestor/memeinvestor_bot/" target="_blank">
+                  <img class="icon-32" src="resources/GitHub-Mark-Light-32px.png"/></a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <ul class="sidenav" id="mobile-sidebar">
+        <li class="sidenav-top">
+         <h5>menu</h5>
+         <a class="sidenav-close" href="#!"><i class="material-icons">close</i></a>
+        </li>
+        <li><a id="without-this-everything-brokes">lol</a></li>
+        <li class="sidebar-search">
+            <input id="investor-username-mobile" placeholder="search user" type="search"  >
+        </li>
+        <!-- <li><a href="foo.html">brokers</a></li> -->
+        <li><a href="index.html">home</a></li>
+        <li><a class="grey-text" >leaderboard</a></li>
+        <li><a href="https://github.com/MemeInvestor/memeinvestor_bot/">
+        <img class="icon-32" src="resources/GitHub-Mark-Light-32px.png" target="_blank"></a></li>
+    </ul>
+    
+    <div id="announcements-modal" class="modal ">
+    </div>
+
+<!-- PUT PAGE CODE HERE -->
+
+    <div class="container" >
+      <div class="section center-align">
+        <h4 class="no-margin-top" >Top Traders wall</h4>
+        <p>Select one of the market seasons below to view it's top 50 traders.</p>
```

Comment:
The purpose of this page is already clear. We can simplify the instructions to "Select a season" or even remove them entirely and just put a label next to the buttons that says, "Season:"